fendermint/app/options/src/log.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022-2024 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use clap::{builder::PossibleValue, ValueEnum};
+
+use lazy_static::lazy_static;
+use tracing_subscriber::EnvFilter;
+
+/// Standard log levels, or something we can pass to <https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html>
+///
+/// To be fair we could pass of everything except "off" as a filter.
```

Comment:
That's not true actually, `EnvFilter` uses a different type which handles "off" too.

Reply:
But I thought I'd leave it like this also to try how to implement a `ValueEnum` manually.